chain/bitcoind_events.go

```diff
@@ -645,13 +646,22 @@ func (b *bitcoindRPCPollingEvents) txEventHandlerRPC() {
 // the chain backend's mempool no longer containing it.
 type mempool struct {
 	sync.RWMutex
+
+	// txs stores the txids in the mempool.
 	txs map[chainhash.Hash]bool
+
+	// inputs stores the inputs of the txids in the mempool. This is
+	// created for fast lookup.
+	//
+	// TODO(yy): create similar maps for outputs.
```

Comment:
Outputs to be used to finding new scripts created in the mempool?

Reply:
yeah, updated the docs